docs/Manual Build Instructions.md

```diff
@@ -0,0 +1,92 @@
+
+The final file structure of the `Built` directory should be as follows:
+- Linux_32_ARM
+    - FXP.llb
+- Linux_x64
+    - FXP.llb
+- Pharlap
+    - FXP.llb
+
+### 2. [Scan Engine Module Libraries](https://github.com/ni/niveristand-scan-engine-module-libraries)
+1. Open `Modules.lvproj`
+1. Build the `Modules` build specification for each target
+1. Build the `NI ECAT Remote IO` build specification for each target
+
+The final file structure of the `Built` directory should be as follows:
+- Linux_32_ARM
+    - Modules.lvlibp
+    - NI ECAT Remote IO.llb
+- Linux_x64
+    - Modules.lvlibp
+    - NI ECAT Remote IO.llb
+- Pharlap
+    - Modules.lvlibp
+    - NI ECAT Remote IO.llb
+
+### 3. [Scan Engine Custom Device](https://github.com/ni/niveristand-scan-engine-ethercat-custom-device)
+
+For each target type (Linux_32_ARM, Linux_x64, and Pharlap):
+
+1. Copy dependencies built from previous repositories' `Built` directory into the `Custom Device Source` directory
+    - FXP.llb
+    - Modules.lvlibp
+    - NI ECAT Remote IO.llb
+1. Open `Scan Engine.lvproj`
+    - Depending on the versions of VeriStand and LabVIEW installed on the development PC, the version of the VeriStand .NET assemblies may mismatch with the project. Use .lvproj.config files to lock the version of the assemblies used by the project, or manually navigate to the assemblies on disk at `C:\Windows\Microsoft.NET\assembly\GAC_MSIL\`.
+    - You may see dialogs concerning a mismatch of target versions when loading LLBs and PPLs for other operating systems. These are safe to ignore.
+1. Build the `Engine Release` build specification
+1. Close LabVIEW completely and repeat for the next target type
+
+For Windows using Pharlap dependencies:
+
+1. Open `Scan Engine.lvproj`
+1. Build the `Configuration Release` build specification
+
+The final file structure of the `Built` directory should be as follows (only two directory levels are shown):
+- Scan Engine
+    - data\
+    - Glyphs\
+    - Linux_32_ARM\
+    - Linux_x64\
+    - Pharlap\
+    - VxWorks\
+    - Custom Device Scan Engine.xml
+    - Scan Engine - Configuration.llb
+- Errors
+    - scanengineveristand-errors.txt
+
+## Installing a manually built Scan Engine and EtherCAT Custom Device
+These instructions are only for manually built instances of the Custom Device. For released versions on GitHub, the NI Package installer performs these actions.
+
+The build specifications for the Custom Device contain post-build action VIs to copy the built files to the final directory. These steps are only for distributing the built files to a different PC.
+
+### Install Custom Device Files
+1. Ensure VeriStand is completely closed
+1. Delete existing contents in the `<Public Documents>\National Instruments\NI VeriStand 20xx\Custom Devices\Scan Engine` directory
+1. Copy the `Built\Scan Engine` directory into the `<Public Documents>\National Instruments\NI VeriStand 20xx\Custom Devices` directory.
```

Comment:
Why does this line end in a period but the others don't?

Reply:
I appreciate this is a question instead of a statement to remove it.
There actually is an answer: I copied this section from the old readme.txt file (that needs deleted), then updated its style/formatting. Missed the period.